### ISSUE TITLE
Footers not skipped in systemctl list-sockets parser

### ISSUE THREAD

**Issue by kellyjonbrazil** [kellyjonbrazil/jc#38]
Footer lines are not being skipped.
```
…
 {
    "listen": "kobject-uevent",
    "unit": "1",
    "activates": "systemd-udevd-kernel.socket"
  },
  {
    "listen": "12",
    "unit": "sockets",
    "activates": "listed."
  },
  {
    "listen": "Pass",
    "unit": "--all",
    "activates": "to"
  }
]
```

**Comment by kellyjonbrazil** [comment]
Looks like the issue is not with the parser, but with the Magic syntax.
```
jc -p systemctl list-sockets
```
Looks like the Magic syntax is picking the wrong parser, probably plain `systemctl`.

**Comment by kellyjonbrazil** [comment]
@philippeitis it looks like this issue was introduced https://github.com/kellyjonbrazil/jc/pull/32.

**Comment by kellyjonbrazil** [comment]
fix merged into v1.8.1
